scripts/init.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -eou pipefail
```

Comment:
see: https://coderwall.com/p/fkfaqq/safer-bash-scripts-with-set-euxo-pipefail for more details on the rationale behind each of the parameters